libs/libarchfpga/src/logic_types.h

```diff
@@ -25,12 +25,20 @@ enum PORTS {
     ERR_PORT
 };
 
+enum TriggeringEdge {
```

Comment:
That's because the places where it was not used were in libtatum. I wanted to keep this library with minimal possible changes. However, now I added a similar enum in libtatum because it is an external lib